.github/workflows/test.yml

```
@@ -335,6 +339,35 @@ jobs:
+        sudo touch "${MNT_LOOP_PATH}"
+        sudo fallocate -z -l "${TMP_LVM_SIZE_BYTES}" "${MNT_LOOP_PATH}"
+        export TMP_LOOP_DEV=$(sudo losetup --find --show "${MNT_LOOP_PATH}")
+        sudo pvcreate -f "${TMP_LOOP_DEV}"
+        VG_NAME="docker"
+        sudo vgcreate "${VG_NAME}" "${TMP_LOOP_DEV}" "${ROOT_LOOP_DEV}"
+        echo "creating docker volume"
+        sudo lvcreate -l 100%FREE -n dockerlv "${VG_NAME}"
+        sudo mkfs.ext4 -Enodiscard -m0 "/dev/mapper/${VG_NAME}-dockerlv"
+        sudo mount "/dev/mapper/${VG_NAME}-dockerlv" "${MOUNT_PATH}"
+        sudo chown -R runner:runner "${MOUNT_PATH}"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)